.github/workflows/windows.yml

```diff
@@ -96,7 +96,7 @@ jobs:
     - name: Build & Install
       shell: cmd
       run: |
-        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\vc\Auxiliary\build\vcvarsall.bat" x64
+        call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
```

Comment:
See https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md, which lists this path